client/app/cloud/project/add/cloud-project-add.service.js

```diff
@@ -0,0 +1,70 @@
+class CloudProjectAddService {
+    constructor ($q, OvhApiCloud, OvhApiMe, OvhApiOrder) {
+        this.$q = $q;
+        this.OvhApiCloud = OvhApiCloud;
+        this.OvhApiMe = OvhApiMe;
+        this.OvhApiOrder = OvhApiOrder;
+    }
+
+    getCloudProjectOrders () {
+        return this.OvhApiCloud.Lexi().order().$promise;
+    }
+
+    getCloudProjectOrder (orderId) {
+        return this.getCloudProjectOrders()
+            .then(response => _.first(_.filter(response, order => order.orderId === orderId)));
+    }
+
+    orderCloudProject (description, voucher) {
+        return this.OvhApiMe.Lexi().get()
+            .$promise
+            .then(user => this.OvhApiOrder.Cart().Lexi().post({}, { ovhSubsidiary: user.ovhSubsidiary }).$promise)
+            .then(cart => this.OvhApiOrder.Cart().Lexi().assign({ cartId: cart.cartId }).$promise.then(() => cart))
+            .then(cart => this.OvhApiOrder.Cart().Product().Lexi().post({ cartId: cart.cartId, productName: "cloud" }, {
+                duration: "P1M",
+                planCode: "project",
+                pricingMode: "default",
+                quantity: 1 }).$promise)
+            .then(response => this._configureOrder(response, description, voucher))
+            .then(response => this.OvhApiOrder.Cart().Lexi().checkout({ cartId: response.cartId }).$promise)
+            .then(response => {
+                if (!response.prices.withTax.value) {
```

Comment:
response.priceWithTax.value

Reply:
I'll check but I choosed to use response.prices.withTax.value because it was working.  I've not seen a priceWithTax propert but I'll check again